### PR TITLE
Add initial field to the helper input_number in UI

### LINKED ISSUE #12214: Helpers lost some values to configure by UI compared to yaml
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I need a few input_number values with a initial value. This can not be set by UI but very well by yaml. I got them by editing around in /.storage/input_number, but that is no real solution. This value is availbale for a few more you now can find in the new helpers section. Maybe these need some love too.

### Describe the behavior you expected

That all values can be edited.

### Steps to reproduce the issue

Just create a new helper like a input_number and try to select the initial value.

### What version of Home Assistant Core has the issue?

2022.4.0b4

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

Any

### Which operating system are you using to run this browser?

Windows 11

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alvinchen1**
## Proposed change

Adding the initial field to the helper - input_number in UI.  Supported in YAML. See https://www.home-assistant.io/integrations/input_number/#initial.  Added localizations of "initial" by copying any instance of counter helper's initial and pasting to input_number.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #12214
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @alvinchen1, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;frontend#13378).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by steverep — CHANGES_REQUESTED** [review]
See translations/README.md.  Only the English translation file in SRC/translations should be modified.

Also, when adding the translation, you should reference the existing translation key directly instead of copying. See the documentation and other examples in the file.

**Comment by alvinchen1** [comment]
Well, that's a heck of a lot easier.  Sorry about that, first time contributor.  Reverted changes to translations and edited only the src translation en.json.

File: src/translations/en.json [hunk 1]
```diff
@@ -916,6 +916,7 @@
           "pattern": "Regex pattern for client-side validation"
         },
         "input_number": {
+          "initial": "Initial value",
```
Comment (zsarnett) [inline src/translations/en.json:919]:
```suggestion
          "initial": "Starting value",
```

**Review by zsarnett — APPROVED** [review]

**Review by zsarnett — APPROVED** [review]

**Review by steverep — APPROVED** [review]

**Pull request merged**